src/main/webapp/WEB-INF/jsp/application/vacation_overview.jsp

```diff
@@ -44,122 +47,233 @@
+                                        <spring:message code="month.all"/>
+                                    </option>
+                                    <option disabled>──────────</option>
+                                    <c:forEach var="i" begin="1" end="12">
+                                    <option value="${i}" ${i == selectedMonth ? 'selected="selected"' : ''}">
+                                        <spring:eval var="monthName" expression="T(org.synyx.urlaubsverwaltung.util.DateUtil).getMonthName(i)" />
+                                        <c:out value="${monthName}"/>
+                                    </option>
+                                    </c:forEach>
+                                </select>
+                            </div>
                         </div>
                     </div>
-                </div>
-                <div class="form-group">
-                    <div class="row">
-                        <label class="control-label col-md-3" for="departmentSelect">
-                            <spring:message code="overview.vacationOverview.department"/>:
-                        </label>
-                        <div class="col-md-6">
-                            <select id="departmentSelect" name="departmentSelect" size="1" path="" class="form-control">
-                                <c:forEach items="${departments}" var="department">
-                                    <option value="${department}">
-                                        <c:out value="${department.name}"/>
-                                    </option>
-                                </c:forEach>
-                            </select>
+                    <div class="form-group">
+                        <div class="row">
+                            <label class="control-label col-md-3" for="departmentSelect">
+                                <spring:message code="overview.vacationOverview.department"/>:
+                            </label>
+                            <div class="col-md-6">
+                                <select id="departmentSelect" name="department" size="1" class="form-control">
+                                    <c:forEach items="${departments}" var="department">
+                                        <option value="${department.name}" ${department.name == selectedDepartment ? 'selected="selected"' : ''}>
+                                            <c:out value="${department.name}"/>
+                                        </option>
+                                    </c:forEach>
+                                </select>
+                            </div>
                         </div>
                     </div>
                 </div>
-            </div>
+            </form>
 
             <div class="row">
                 <div class="col-xs-12">
-                    <hr/>
-                    <div id="vacationOverview"></div>
+                    <hr class="print:hidden" />
```

Comment:
Does not work in Chrome and Firefox, in other JSPs is `hidden-print` used.

Reply:
I've printed it. it works.

Reply:
Working on Windows, too